Makefile

```
@@ -255,7 +255,7 @@ index.html: blog/announcements-rss.xml blog/index.html
 community/commercial-support.html: community/commercial-support.html.in
 
 community/commercial-support.html.in: community/commercial-support.toml
-	shuffle-commercial-providers --input community/commercial-support.toml > $@
+	update-commercial-providers --input community/commercial-support.toml > $@
```

It should be possible to just have

```suggestion
community/commercial-support.html: community/commercial-support.toml
	update-commercial-providers --input community/commercial-support.toml > $@
```

and remove the `.in` file from git.

---

Removed.

---

Removing `community/commercial-support.html.in` is not a good idea since it is included in `
community/commercial-support.html`.

---

It is just an intermediary step. The suggested change above should make it unnecessary.